crate/cli/src/actions/findex/insert_or_delete.rs

```diff
@@ -42,11 +42,11 @@ impl InsertOrDeleteAction {
     /// - The Findex instance cannot insert or delete the data.
     /// - The semaphore cannot acquire a permit.
     async fn insert_or_delete(
-        &self,
-        rest_client: &FindexRestClient,
+        self,
+        rest_client: FindexRestClient,
         is_insert: bool,
-    ) -> CliResult<Keywords> {
-        let file = File::open(self.csv.clone())?;
+    ) -> CliResult<String> {
```

Comment:
as discussed, we need a readable CLI. I suggest to revert this change, let Keywords as return type and keep this `cloned()` ligne 76 `let written_keywords = bindings.keys().cloned().collect::<Vec<_>>();`